### ISSUE TITLE
Change Default Behavior of "fix_com" and "fix_orientation"

### ISSUE THREAD

**Issue by coltonbh** [MolSSI/QCElemental#336]
**Describe the bug**
I've been bitten by this multiple times. I forget that unless I explicitly tell QCElemental to _not_ rotate or translate my molecule, it does. And then I wonder why there's no gradient in the z-axis, or something like this. 

I think a better default is to _not_ change people Molecule objects unsuspectingly unless they explicitly tell QCEl to do it. How about swapping out these defaults to True?

Code here: https://github.com/MolSSI/QCElemental/blob/8e5a8cff52a6438ff9d6c1c6bbf1aeb4f02f12e1/qcelemental/models/molecule.py#L263-L282

**To Reproduce**
Pass any .xyz file or geometry to Molecule. Get surprised when you suddenly have a different object on your hands with a different geometry! Or worse, do calculations and get confused about why gradients are suddenly missing components along the z-axis!

**Expected behavior**
If I pass a geometry and don't tell QCElemental to mess with it, I expect that Molecule object to faithfully represent the data I passed it upon instantiation. If I explicitly tell QCel to translate and rotate my object, then I'd expect it to do so.

**Additional context**
If you agree with changing this default behavior let me know and I'll create a PR.

**Comment by loriab** [comment]
I agree that it can be surprising -- I've gotten burned myself. A lot of QC programs do the same by treating input cartesians as eligible for reorientation. I think it's a case of either default will surprise in a good fraction of cases, so best to leave as-is. It wouldn't be too hard to switch here, but a lot of QCEngine logic would change.

That said, the description you quoted about QCEl pre-processing the Mol doesn't ring quite true to me (though I could be mistaken). It sounds more like what QCPortal might do rather than the Mol constructor alone.

**Comment by coltonbh** [comment]
For my own edification, why do "a lot of QC programs do the same by treating input cartesians as eligible for reorientation"? I don't actually know why this would be considered desirable--what's the intention behind doing this?

**Comment by bennybp** [comment]
It's a lot easier to detect and exploit symmetry when molecules are in the "standard orientation", which reduces cost. It also just generally makes things consistent (in the standard orientation, principal/highest order rotation axis gets aligned with the Z-axis).

See for example, Gaussian: https://gaussian.com/symmetry/

**Comment by loriab** [comment]
Another thing to watch out for is that some QC programs shift to center of mass, and some shift to center of charge (so that isotope substitutions use the same frame).

**Comment by coltonbh** [comment]
Yikes! Thanks for the heads up!

If using QCEngine and I set `fix_com` and `fix_orientation` to `True` will this prevent all the various harnessed programs from doing these things?

**Comment by loriab** [comment]
> If using QCEngine and I set fix_com and fix_orientation to True will this prevent all the various harnessed programs from doing these things?

It's supposed to, but testing rolled out after several of the harnesses, so I can only vouch for the ones I worked on and tested: https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/tests/test_alignment.py#L88-L95
